js/details.js

```diff
@@ -1,3 +1,43 @@
+// Splash
+var splash = `
+                                                                     ..;===+.
+                                                                  .:=iiiiii=+=
+                                                               .=i))=;::+)i=+,
+                                                            ,=i);)I)))I):=i=;
+                                                         .=i==))))ii)))I:i++
+                                                       +)+))iiiiiiii))I=i+:\'
+                                  .,:;;++++++;:,.       )iii+:::;iii))+i=\'
+                               .:;++=iiiiiiiiii=++;.    =::,,,:::=i));=+\'
+                             ,;+==ii)))))))))))ii==+;,      ,,,:=i))+=:
+                           ,;+=ii))))))IIIIII))))ii===;.    ,,:=i)=i+
+                          ;+=ii)))IIIIITIIIIII))))iiii=+,   ,:=));=,
+                        ,+=i))IIIIIITTTTTITIIIIII)))I)i=+,,:+i)=i+
+                       ,+i))IIIIIITTTTTTTTTTTTI))IIII))i=::i))i=\'
+                      ,=i))IIIIITLLTTTTTTTTTTIITTTTIII)+;+i)+i\`
+                      =i))IIITTLTLTTTTTTTTTIITTLLTTTII+:i)ii:\'
+                     +i))IITTTLLLTTTTTTTTTTTTLLLTTTT+:i)))=,
+                     =))ITTTTTTTTTTTLTTTTTTLLLLLLTi:=)IIiii;
+                    .i)IIITTTTTTTTLTTTITLLLLLLLT);=)I)))))i;
+                    :))IIITTTTTLTTTTTTLLHLLLLL);=)II)IIIIi=:
+                    :i)IIITTTTTTTTTLLLHLLHLL)+=)II)ITTTI)i=
+                    .i)IIITTTTITTLLLHHLLLL);=)II)ITTTTII)i+
+                    =i)IIIIIITTLLLLLLHLL=:i)II)TTTTTTIII)i\'
+                  +i)i)))IITTLLLLLLLLT=:i)II)TTTTLTTIII)i;
+                +ii)i:)IITTLLTLLLLT=;+i)I)ITTTTLTTTII))i;
+               =;)i=:,=)ITTTTLTTI=:i))I)TTTLLLTTTTTII)i;
+             +i)ii::,  +)IIITI+:+i)I))TTTTLLTTTTTII))=,
+           :=;)i=:,,    ,i++::i))I)ITTTTTTTTTTIIII)=+\'
+         .+ii)i=::,,   ,,::=i)))iIITTTTTTTTIIIII)=+
+        ,==)ii=;:,,,,:::=ii)i)iIIIITIIITIIII))i+:\'
+       +=:))i==;:::;=iii)+)=  \`:i)))IIIII)ii+\'
+     .+=:))iiiiiiii)))+ii;
+    .+=;))iiiiii)));ii+
+   .+=i:)))))))=+ii+
+  .;==i+::::=)i=;
+  ,+==iiiiii+,
+  \`+=+++;\`
+`;
+
```

Comment:
Hey David, thanks for the contributions! I didn't get notifications about the pull request.

Only problem I have is with this multiline string. Would you be able to change it to [this style](https://google.github.io/styleguide/javascriptguide.xml?showone=Multiline_string_literals#Multiline_string_literals)? I would prefer not to use `` style strings as ECMAScript 6 support is still lacking in some browsers.